src/components/block/index.ts

```diff
@@ -994,4 +995,11 @@ export default class Block extends EventsDispatcher<BlockEvents> {
   private dropInputsCache(): void {
     this.cachedInputs = [];
   }
+
+  /**
+   * Get the list of mergeable blocks with the current block
+   */
+  public get mergeableWithBlocks(): string[]|undefined {
+    return this.tool.mergeableBlocks;
```

Comment:
one tool should not know anything about another tool by design. This feature could be implemented using existing `conversionConfig`